### DISCUSSION TITLE
Multilingual model V4

### DISCUSSION THREAD

**Discussion by pd-intra** [PaddlePaddle/PaddleOCR#12888] [Ideas & Features]
Hello,

First of all, thank all of you for your amazing work. PaddleOCR is clearly the best OCR i've ever had to test.
Recently, you have release pp-ocrv4 rec for chinese and english, det for chinese.
In your presentation of the v4, we can see some information about pp-ocrv4-mul, your ocr for multilingual recognition.
DId you know when we will have access to this model and if he's already release, where can i find it ?

Thanks in advance

**Comment by ghassenBtb** [comment]
Hello,

Same question here.
It would be great if you could share with us the PpocrV4 Multilingual Detection Model  :)
